main.js

```diff
@@ -201,6 +202,72 @@ function clickHandlerHack() {
 // but SillyTavern's complete lack of encapsulation makes it hard to find a clean solution.
 hljs.registerLanguage("sorcery-stscript", () => hljs.getLanguage("stscript"));
 
+// Enable a script by ID
+function enableScript(_, scriptId) {
+    if (!scriptId) {
+        toastr.warning("Please provide a script ID to enable");
+        return;
+    }
+       
+    const id = parseInt(scriptId);
+    if (isNaN(id)) {
+        toastr.error("Script ID must be a number");
+        return;
+    }
+    
+    const script = settings.scripts.find(s => s.id === id);
+    
+    if (!script) {
+        toastr.error(`No script found with ID ${id}`);
+        return;
+    }
+    
+    script.enabled = true;
+    
+    // Update the visual toggle if the script is currently rendered using data attributes
+    $(`.sorcery-id[data-script-id="${id}"]`).closest('.world_entry').find('.sorcery-enabled')
+        .removeClass('fa-toggle-off')
+        .addClass('fa-toggle-on');
+    
+    toastr.success(`Script with ID ${id} has been enabled`);
+    saveSettingsDebounced();
+};
+
+// Disable a script by ID
+function disableScript(_, scriptId) {
+    if (!scriptId) {
+        toastr.warning("Please provide a script ID to disable");
+        return;
+    }
+       
+    const id = parseInt(scriptId);
+    if (isNaN(id)) {
+        toastr.error("Script ID must be a number");
+        return;
+    }
+    
+    const script = settings.scripts.find(s => s.id === id);
+    
+    if (!script) {
+        toastr.error(`No script found with ID ${id}`);
+        return;
+    }
+    
+    script.enabled = false;
+    
+    // Update the visual toggle if the script is currently rendered using data attributes
+    $(`.sorcery-id[data-script-id="${id}"]`).closest('.world_entry').find('.sorcery-enabled')
+        .removeClass('fa-toggle-on')
+        .addClass('fa-toggle-off');
+    
+    toastr.success(`Script with ID ${id} has been disabled`);
+    saveSettingsDebounced();
+};
+
+// Register the slash commands
```

Comment:
Comments should explain what the code does, not repeat the code. "Register the slash commands" followed by `registerSlashCommand` is just noise.